roles/openshift_node/tasks/main.yml

```diff
@@ -23,7 +23,6 @@
       iptables_sync_period: "{{ openshift_node_iptables_sync_period | default(None) }}"
       kubelet_args: "{{ openshift_node_kubelet_args | default(None) }}"
       labels: "{{ lookup('oo_option', 'openshift_node_labels') | default( openshift_node_labels | default(none), true) }}"
-      portal_net: "{{ openshift_master_portal_net | default(None) }}"
```

Comment:
+1 on just persisting the kube service ip, since it will also provide the default for the dns_ip (which we are currently only persisting if the user overrides it, otherwise we are performing a bit of acrobatics to determine it.